src/environments/environment.staging.ts

```diff
@@ -19,6 +19,7 @@ export const environment = {
   production: true,
   identityServerUrl: "https://staging-dex-identity.kn01.fhict.nl",
   apiUrl: "https://staging-dex-api.kn01.fhict.nl",
```

Comment:
```suggestion
  apiUrl: "https://api.staging.dex.software",
```
We now have a new domain